platform/android/audio_driver_jandroid.cpp

```diff
@@ -76,9 +76,9 @@ Error AudioDriverAndroid::init() {
 	//        __android_log_print(ANDROID_LOG_VERBOSE, "SDL", "SDL audio: opening device");
 
 	JNIEnv *env = ThreadAndroid::get_env();
-	int mix_rate = GLOBAL_DEF_RST("audio/mix_rate", 44100);
+	mix_rate = GLOBAL_DEF_RST("audio/mix_rate", DEFAULT_MIX_RATE);
```

Comment:
We were declaring a local here, so never updating the `mix_rate` class member AFAICT.
I think this driver is not used at all though, so was likely not an issue :)

Reply:
Yes, this driver is not used at all, maybe we should drop it? or is there a reason to keep it?

Reply:
I'm fine with dropping it, I'll do another PR for that.

Reply:
Actually it's pretty weird, `AudioDriverAndroid` *is* used in `java_glue.cpp`... That's very fishy.

Reply:
Those calls in `java_glue.cpp` can be dropped if we remove `AudioDriverAndroid`. And also the audio code in `GodotIO.java` can be dropped.

Reply:
Can I let you do it? I think you're more familiar with it than I am :)

Reply:
Sure :)

Reply:
Reverted those changes for now, I'll open an issue about dropping `AudioDriverAndroid`.